### ISSUE TITLE
Is there any Org ID for NIH data?

### ISSUE THREAD

**Issue by aaryani** [researchgraph/Crosswalks#74]

**Comment by BenZapilko** [comment]
Yes, there is the field ORG_IPF_CODE. It includes an IPF code for the organization. If it is not available in the mapped data, we need to include it into the mapping.

More details on IPF can be found here:
https://public.era.nih.gov/chl/public/search/ipfNumberByOrgName.era

**Comment by aaryani** [comment]
Then we should consider creating Organisation nodes from ORG_IPF_CODE
